internal/provider/slos_data_source_test.go

```diff
@@ -111,7 +111,6 @@ data "honeycombio_slos" "exact" {
 }
 `, dataset, testPrefix),
 				Check: resource.ComposeAggregateTestCheckFunc(
-					resource.TestCheckResourceAttr("data.honeycombio_slos.all", "ids.#", "5"),
```

Comment:
This test had been flakey since the Boards testsuite starting making additional SLOs. 

I've just removed it for the time being as the other tests do a decent job ensuring the datasource is working.